### ISSUE TITLE
Define more En-route radar heads

### ISSUE THREAD

**Issue by luke11brown** [luke11brown/UK-TopSky-Controller-Pack#1]
Currently only 3 radar heads defined, define the rest of the NERL sites

**Comment by sw104** [comment]
Is this for inclusion in the EuroScope ESE? I already have some data in the correct format if it is:

[ESE Radar Data.txt](https://github.com/luke11brown/UK-TopSky-Controller-Pack/files/4918263/ESE.Radar.Data.txt)

The file also contains definitions for some radar holes based on NATS self-assesment of their coverage which was in some KMZ files on their site a while back.

It may be a bit outdated now, so I may have a look to see if I can confirm any of the details or properly reference some sources.

**Comment by sw104** [comment]
Looks like the self-assessment maps for wind farm devlopment (which the radar holes are based on) can be found [https://www.nats.aero/services-products/catalogue/n/wind-farms-self-assessment-maps/](here).

It looks like it will also be possible to get more accurate [cone slope](https://www.euroscope.hu/wp/ese-files-description/) values via some of the radar line of sight assessments e.g. for [Lowther Hill](http://marine.gov.scot/sites/default/files/lf009-env-ma-rpt-0042_eia_report_vol_3_app_14f_los_lowther.pdf). There also seems to be more accurate range estimations in some of the other assessments e.g. [this one](http://marine.gov.scot/sites/default/files/lf009-env-ma-rpt-0042_eia_report_vol_3_app_14f_los_lowther.pdf) has range info for the Debden and Cromer PSRs (as well as the EGMC PSR).

**Comment by luke11brown** [comment]
Nice!

The TopSky PSR raw video format is defined in the developer guide section 2.8

**Comment by luke11brown** [comment]
Nonetheless, I'd like to include the ESE RADAR2 defintions in this pack.

**Comment by sw104** [comment]
Great! I'll start trying to gather the required data and make sure what I have is accurate.

**Comment by sw104** [comment]
I've created a [separate repo](https://github.com/sw104/UK-Civil-Radars) which contains the raw data I have collected about each site. All of the data in this repo has an attributed source (so far), but I will also indicate where assumptions have been made. It's not quite at parity with the original data I collected, but it shouldn't take too long to get there.

After the data I gather is of sufficient quality I'll format it appropriately for the EuroScope ESE file definitions and TopSky.
